openmc/element.py

```diff
@@ -181,6 +236,70 @@ def expand(self, percent, percent_type, enrichment=None,
             for nuclide in abundances.keys():
                 abundances[nuclide] /= sum_abundances
 
+        # Modify mole fractions if enrichment provided
+        # New treatment for arbitrary element
+        # Interpret required enrichment as weight
+        elif enrichment is not None and enrichment_target is not None:
+
+            # Check if is a single isotope mixture
```

Comment:
Is the comment correct here? Should this say "# Check if it is a two-isotope mixture"?